.travis.yml

```diff
@@ -1,11 +1,21 @@
 language: node_js
 node_js:
-  - "5"
```

Comment:
The spacing here was automatically fixed when I ran a Travis command on my terminal